include/linker/linker-defs.h

```diff
@@ -249,6 +249,11 @@ extern char _vector_end[];
 /* end address of image, used by newlib for the heap */
 extern char _end[];
 
+/* ramfunc sections */
```

Comment:
Can you add some explanatory text, for example, as the one for _app_ram_start (lines 184 - 190)?